pvlib/pvsystem.py

```diff
@@ -333,30 +334,33 @@ def get_aoi(self, solar_zenith, solar_azimuth):
 
     @_unwrap_single_value
     def get_irradiance(self, solar_zenith, solar_azimuth, dni, ghi, dhi,
-                       dni_extra=None, airmass=None, model='haydavies',
-                       **kwargs):
+                       albedo=None, dni_extra=None, airmass=None,
+                       model='haydavies', **kwargs):
```

Comment:
Anyone passing kwargs as args will see their code break due to the new kwarg in the middle. We've historically not promised that kind of API compatibility so it's fine with me. And I like this signature better than the alternative.

Reply:
Ha, and I read a little further and see that we're hitting this on line -386/+397!

Reply:
At the very least I think we should match the `irradiance.get_total_irradiance` order: dni_extra, airmass, albedo